src/mslice/workspace/helperfunctions.py

```diff
@@ -116,3 +115,60 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         if self.workspace:
             self.workspace.remove_saved_attributes()
         return True
+
+
+class WorkspaceNameHandler:
+
+    def __init__(self, ws_name: str):
+        self.ws_name = ws_name
+
+    def add_prefix(self, prefix) -> str:
+        return prefix + self.ws_name
+
+    def add_sufix(self, sufix) -> str:
+        return self.ws_name + sufix
+
+    def scaled(self, scaling_factor: float) -> str:
+        return self.add_sufix("_ssf_" + f"{scaling_factor:.2f}".replace('.', '_'))
+
+    def subtracted(self, scaling_factor: float) -> str:
+        return self.add_sufix("_minus_ssf_" + f"{scaling_factor:.2f}".replace('.', '_'))
+
+    def summed(self) -> str:
+        return self.add_sufix("_sum")
+
+    def rebosed(self) -> str:
+        return self.add_sufix('_bosed')
+
+    def combined(self) -> str:
+        return self.add_sufix('_combined')
+
+    def merged(self) -> str:
+        return self.add_sufix('_merged')
+
+    def isHiddenFromMsl(self) -> bool:
```

Comment:
You've swapped from snake case to camel case here, worth having a flick through these if you haven't already: https://developer.mantidproject.org/Standards/PythonStandards.html